build.yaml

```diff
@@ -282,8 +282,8 @@ common_cli:
 job_type:
   run_trial:
   # http://build.clusterhq.com/builders/flocker-centos-7
-    run_trial_on_AWS_CentOS_7:
-      on_nodes_with_labels: 'aws-centos-7-T2Small'
+    run_trial_on_AWS_CentOS_7_with_SELinux:
```

Comment:
:question: Given that you are just changing the images used for the build, is there any reason to also rename the builds? It seems just like a new configuration of the environment, rather than a different builder.